focus/operations/src/status.rs

```diff
@@ -16,6 +16,16 @@ pub fn run(
     let repo = Repo::open(sparse_repo.as_ref(), app)?;
     let selections = repo.selection_manager()?;
     let selection = selections.selection()?;
+    let is_filter_view = repo.working_tree().unwrap().get_filter_config()?;
+
+    println!();
+    if is_filter_view {
+        println!("Filtered view. Only selections are present in the worktree.");
+    } else {
+        println!("Unfiltered view. All repo contents are present in the worktree.")
+    }
+    println!();
```

Comment:
Let's change these to `eprintln` in case people are processing the selection through further redirections...

Reply:
fixed this.